azure-pipelines.yml

```diff
@@ -251,20 +251,16 @@ stages:
           & dotnet build $testProjFileName -c $(BuildConfiguration)
           Test-ExitCode "ERROR: UnitTest project build FAILED."
 
-          Write-Host "List Sonar*.dll files"
-          Get-ChildItem -Recurse .\tests\SonarAnalyzer.UnitTest\bin\${BuildConfiguration}\ -Filter Sonar*.dll
+          Write-Host "Register OpenCover profilers. This step needs to be done manually on CI agents. See https://github.com/OpenCover/opencover/raw/master/main/OpenCover.Documentation/Usage.pdf"
+          regsvr32 $(NUGET_PACKAGES)\opencover\4.7.922\tools\x86\OpenCover.Profiler.dll
```

Comment:
The primary way `-register:user` option doesn't work?
I'd prefer OpenCover to do the dirty jobs by itself. Otherwise, do we know if both 32 and 64 are needed and if/when we can remove 32 for example?

Reply:
Yes, `-register:user` does not work for .Net Core and .Net

Reply:
The options are `x86` and `x64`. I will try to see if `x64` is enough.

Reply:
Unfortunately we need both. `x86` version is used for .Net framwework tests and `x64` is used for .Net 5